content/en/platform/corda/4.4/enterprise/cordapps/cordapp-build-systems.md

```diff
@@ -185,6 +185,22 @@ in the CorDapp JAR
 
 Note that the `cordformation` and `cordapp` Gradle plugins can be used together.
 
+### Migrating to the latest version of Corda Gradle plugins
+
+{{< note >}}
+The latest version of Corda Gradle plugins is 5.1.x, which require Gradle 7.
```

Comment:
Is it a single plugin or multiple plugins?

Reply:
There are multiple Corda Gradle plugins:
- `cordapp` to build a CorDapp
- `cordformation` to deploy CorDapps into a local Corda installation (for test purposes)
- `quasar-utils` to apply Quasar to Gradle's `Test` and `JavaExec` tasks.